pkg/controller.v1/tensorflow/pod.go

```diff
@@ -240,6 +241,22 @@ func (tc *TFController) createNewPod(tfjob *tfv1.TFJob, rt, index string, spec *
 	return nil
 }
 
+func checkPodTTL(tfjob *tfv1.TFJob) bool {
+	if value, ok := tfjob.Annotations[TFJobWaitingWorkerAnnotation]; ok && value != "" {
+		podTTLAfterFinished, err := strconv.ParseInt(value, 10, 64)
+		if err != nil {
+			return true
+		}
+
+		// determine whether the set TTL time is reached.
+		if time.Now().Sub(tfjob.Status.CompletionTime.Time) >= time.Duration(podTTLAfterFinished)*time.Second {
+			return true
```

Comment:
Please add some logging to show why it turned to true or still false.

Reply:
Done